lib/attrsets.nix

```diff
@@ -496,6 +496,10 @@ rec {
   getDev = getOutput "dev";
   getMan = getOutput "man";
 
+  /* Get a package's main executable's path */
```

Comment:
```suggestion
  /* Get path to main executable of a package */
```

Reply:
@samuelgrf The suggested change seems like unidiomatic English to me, but I just pushed an alternative formulation without nested `'s`.  Can you confirm this solves the issue you saw, or clarify what was the issue?

Reply:
I guess something about the nested `'s` just didn't sound right to me.
This seems more like a personal preference now that I think about it, you can change it back if you want.

The alternative formulation sounds more natural than my suggestion, so I'll mark this as resolved.

Thanks for your time!

Reply:
You are welcome, and thanks for pointing out the comment didn't sound very natural  :)